tolumide-ng/EPIC_Mail
# #165161093 User can View Specific Group and Password Reset Feature

#### What does this PR do?
User can view specific group

#### Description of Task to be completed?
- user can view a specific group and edit group name
- user can add new members to a group and delete existing members
- improves the user-interface of the groups page and enable mobile compatibility
- enables mobile compatibility for the messages page
- refactors the add member controller on the groups page
- user can send broadcast message to all members of the group
- Installs nodemailer to enable user password reset
- Improvse UI of password reset page to display modal as response on user action
- Adds more test to increase coverage and solve bug on edit group name validator

#### How should this be manually tested?
Clone the repo, cd into it and run **npm run dev-start**, login to the application or signup if you do not already have an account, and click on the **GROUPS** menu on the navigation bar, try to create and edit the group name or add specific members to the group.
If user forgets there password, then a user can use the password reset feature, please check the secondary email you used to register for epicmail for your new password

#### Any background context you want to provide?
Ensure that all required data are provided to avoid validation error

#### What are the relevant pivotal tracker stories?
#165161093
